Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Inspectors/Collections/BaseCollectionEditor.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using UnityEditor;
+using Microsoft.MixedReality.Toolkit.SDK.UX.Collections;
+
+namespace Microsoft.MixedReality.Toolkit.SDK.UX.Inspectors.Collections
+{
+    [CustomEditor( typeof(BaseObjectCollection), true )]
+    public class CollectionEditor : Editor
+    {
+        public override void OnInspectorGUI()
+        {
+            // Draw the default
+            base.OnInspectorGUI();
```

Comment:
Let's go ahead and grab all the properties and show them here so we can a nice clean inspector.

Reply:
I'm not sure what you mean by this? I don't typically do custom inspectors.

Reply:
If you'd like I can open a PR with the suggested changes. Shouldn't take but a few minutes.

Reply:
I think it is much better to rely on the base's default behavior. By overriding that behavior and rebuilding it here, we just make it harder to add properties to the class (e.g, they now need to be added in two places to show up in the inspector). From past discussions, I think @keveleigh is of the same opinion.

Reply:
Let's not be lazy and do it right.

Reply:
Sure. whatever is easiest, is there an existing example somewhere in MRTK?

Reply:
What I'm asserting is that the base.OnInspectorGUI() or DrawDefaultInspector() methods is "doing it right". It removes code duplication and creates a clearer and easier to maintain inspector for future developers.

Think of it in a more generalized way: say we had a class that provided a virtual DrawMe() function, and then we got a PR to bring in a child class that overrode DrawMe() by copy pasting the body of the parent class's DrawMe() and then added a text label at the bottom. We'd ask for it to be refactored to a call to the parent function followed by adding a the text label.

This is the same, we're inheriting behavior from the default inspector. It's bad practice to duplicate that behavior.